image-layout.md

```diff
@@ -134,7 +134,7 @@ The `imageLayoutVersion` value will align with the OCI Image Specification versi
 
 ### oci-layout Example
 
-```json
+```json,title=OCI%20Layout&mediatype=oci%2Dlayout
```

Comment:
> > I'd still rather have “oci-layout file” (or similar) as the title and a real media type.
>
> it likely make no much sense. Refer to manifest.md or so.

`manifest.md` has `title=Manifest`, which makes sense, because the JSON it's titling is a manifest.  Here, the JSON we're titling is an oci-layout file, so I think we want `title=oci-layout%20file`.